_data/committee2017.yml

```diff
@@ -0,0 +1,23 @@
+- name: Tom
```

Comment:
Maybe reorganise the committee.yml data file? We could have a block for active committee and a block for 2k17 a block for 2k16 etc... Otherwise many data files will exist and thats confusing and unorganised